packages/cli-kit/src/node/ruby.ts

```diff
@@ -15,25 +15,32 @@ const MinBundlerVersion = '2.3.8'
 const MinRubyVersion = '2.3.0'
 const MinRubyGemVersion = '2.5.0'
 
+interface ExecCLI2Options {
+  // Contains token and store to pass to CLI 2.0, which will be set as environment variables
+  adminSession?: AdminSession
+  // Directory in which to execute the command. Otherwise the current directory will be used.
+  directory?: string
+}
 /**
  * Execute CLI 2.0 commands.
  * Installs a version of RubyCLI as a vendor dependency in a hidden folder in the system.
  * User must have a valid ruby+bundler environment to run any command.
  *
  * @param args {string[]} List of argumets to execute. (ex: ['theme', 'pull'])
- * @param adminSession {AdminSession} Contains token and store to pass to CLI 2.0, which will be set as environment variables
+ * @param options {ExecCLI2Options}
  */
-export async function execCLI2(args: string[], adminSession?: AdminSession) {
+export async function execCLI2(args: string[], {adminSession, directory}: ExecCLI2Options = {}) {
```

Comment:
Note that we're going from a positional argument to keyword arguments. Thankfully TS has our back here! All the changes should be present below.